packages/protocol/src/data/inscription.ts

```diff
@@ -21,7 +22,7 @@ export type HttpMethod = "GET" | "POST" | "PUT" | "HEAD" | "DELETE" | "PATCH" |
 export type InputType = "ENTITY" | "FORM" | "RAW";
 export type WsAuth = "NONE" | "WS_SECURITY" | "HTTP_BASIC";
 export type Severity = "INFO" | "WARNING" | "ERROR";
-export type Type = "START" | "INTERMEDIATE" | "ACTIVITY";
+export type Type2 = "START" | "INTERMEDIATE" | "ACTIVITY";
```

Comment:
ugly name... can't we improve this @ivy-rew?

Reply:
yes indeed; we should change that type-info in the backend ... by renaming our new object to "CategoryType"

Reply:
yes, I can do that👍